src/components/ReqCourse.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div
+    :class="{ 'reqcourse--min': compact, completedReqCourse: isCompletedReqCourse }"
+    class="reqcourse"
+    :style="borderColorCSSvar"
+  >
+    <div
+      v-if="!isCompletedReqCourse"
+      class="reqcourse-color"
+      :style="cssVars"
+      :class="{ 'reqcourse-color--min': compact }"
+    >
+      <img src="@/assets/images/dots/sixDots.svg" alt="dots" />
+    </div>
+    <div :class="{ 'reqcourse-content--min': compact }" class="reqcourse-content">
+      <div :class="{ 'reqcourse-main--min': compact }" class="reqcourse-main">
+        <div :class="{ 'reqcourse-top--min': compact }" class="reqcourse-top">
+          <div :class="{ 'reqcourse-code--min': compact }" class="reqcourse-code">
+            {{ courseCodeLabel }}
+          </div>
+        </div>
+        <!-- TODO: Revisit if we want this component for Incomplete Req Courses-->
+        <!-- <div v-if="!compact" class="reqcourse-name">{{ name }}</div>
+        <div class="reqcourse-info">
+          <span v-if="!compact" class="reqcourse-credits">{{ creditString }}</span>
+          <span v-if="!compact && semesterString" class="reqcourse-semesters">{{
+            semesterString
+          }}</span>
+        </div> -->
```

Comment:
I'm not a big fan of commented code, especially when it's not substantial for any reason. Won't stop you from merging, but you should consider removing it